src/main/java/calculator/model/DelimiterCalculator.java

```diff
@@ -0,0 +1,133 @@
+package calculator.model;
+
+import static calculator.exception.ErrorMessage.CONSECUTIVE_DELIMITERS;
+import static calculator.exception.ErrorMessage.INVALID_INPUT;
+import static calculator.exception.ErrorMessage.NEGATIVE_NUMBER;
+import static calculator.exception.ErrorMessage.NULL_OR_EMPTY_INPUT;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DelimiterCalculator {
+
+    private static final String DELIMITER_REGEX = "[,:]";
+    private static final Pattern INVALID_DEFAULT_INPUT_PATTERN = Pattern.compile("[^0-9,:]");
+    private static final Pattern CONSECUTIVE_DELIMITER_PATTERN = Pattern.compile("(,|:){2,}");
+
+    /**
+     * 사용자의 입력 값을 기반으로 계산
+     *
+     * @param input 사용자의 입력 값
+     * @return 계산된 결과 값
+     * @throws IllegalArgumentException : NULL_OR_EMPTY_INPUT
+     */
+    public int calculate(String input) {
+        if (input == null || input.isEmpty()) {
+            throw new IllegalArgumentException(NULL_OR_EMPTY_INPUT);
+        }
+
+        input = input.replace(" ", "");
+
+        if (isUsingDefaultDelimiter(input)) {
+            validateDefaultInput(input);
+        } else {
+            validateCustomInput(input);
+        }
+
+        String[] parts = splitInput(input);
+
+        return Arrays.stream(parts)
+                .mapToInt(this::parseNumber)
+                .sum();
+    }
+
+    /**
+     * 기본 구분자 사용 여부 판단
+     *
+     * @param input 사용자의 입력 값
+     * @return 기본 구분자 사용 여부
+     */
+    private boolean isUsingDefaultDelimiter(String input) {
+        return !input.startsWith("//");
+    }
+
+    /**
+     * 기본 구분자를 사용하는 입력 값 검증 : 숫자와 기본 구분자로만 이루어진 문자열인지, 구분자를 연속으로 사용하지 않는지 판단
+     *
+     * @param input 사용자의 입력 값
+     * @throws IllegalArgumentException : INVALID_INPUT, CONSECUTIVE_DELIMITERS
+     */
+    private void validateDefaultInput(String input) {
+        if (INVALID_DEFAULT_INPUT_PATTERN.matcher(input).find()) {
+            throw new IllegalArgumentException(INVALID_INPUT);
+        }
+
+        if (CONSECUTIVE_DELIMITER_PATTERN.matcher(input).find()) {
```

Comment:
이 부분도 개행을 해주면 좋을 것 같습니다!
```suggestion
        if (CONSECUTIVE_DELIMITER_PATTERN.matcher(input)
                    .find()) {
```

Reply:
감사합니다!